x-pack/plugins/global_search_bar/public/plugin.tsx

```diff
@@ -22,6 +29,12 @@ export interface GlobalSearchBarPluginStartDeps {
 }
 
 export class GlobalSearchBarPlugin implements Plugin<{}, {}, {}, GlobalSearchBarPluginStartDeps> {
+  private config: GlobalSearchBarConfigType;
+
+  constructor(private readonly initializerContext: PluginInitializerContext) {
```

Comment:
The `initializerContext` value isn't used later in this class, so this doesn't need to be a private field. The constructor can be simplified as:
```suggestion
  constructor(initializerContext: PluginInitializerContext) {
```

Reply:
We now have eslint rules that require the access modifier is specified, and especially that it's not used is why I choose to make it private.